test/CollapseTable.js

```diff
@@ -1,10 +1,10 @@
-import applib from '../build/applib'
 import assert from 'assert'
 import domino from 'domino'
+import pagelib from '../build/wikimedia-page-library'
```

Comment:
Let me know if `pagelib` works for you! `wikimediaPageLibrary.module.method` felt too verbose for code.